### ISSUE TITLE
badtime is cool, but it doesn't support "badtime x.go" form.

### ISSUE THREAD

**Issue by go101** [m3db/build-tools#9]
though "badtime ./..." works.

**Comment by richardartoul** [comment]
Yep that's true, the library I'm using works off of import paths so I think it can only work whole packages at a time (which makes sense because it has to resolve types)
